src/python/dxpy/scripts/dx.py

```diff
@@ -4505,6 +4547,11 @@ def register_parser(parser, subparsers_action=None, categories=('other', ), add_
 parser_run.add_argument('--debug-on', action='append', choices=['AppError', 'AppInternalError', 'ExecutionError', 'All'],
                         help=fill("Configure the job to hold for debugging when any of the listed errors occur",
                                   width_adjustment=-24))
+parser_run.add_argument('--batch-csv', dest='batch_csv', metavar="FILE",
+                        help=fill('A file in comma separated value (csv) format, with a subset ' +
```

Comment:
Should say tsv (as well as further variable names), since the delimiter is a tab in the code, unless we want to expose the delimiter.